pkg/minikube/drivers/hyperv/windows.go

```diff
@@ -0,0 +1,137 @@
+// +build windows
+
+/*
+Copyright 2018 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hyperv
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"github.com/golang/glog"
+	"k8s.io/minikube/pkg/minikube/exit"
+	"os/exec"
+	"strings"
+)
+
+var powershell string
+
+var (
+	ErrPowerShellNotFound = errors.New("Powershell was not found in the path")
+	ErrNotAdministrator   = errors.New("Hyper-v commands have to be run as an Administrator")
+	ErrNotInstalled       = errors.New("Hyper-V PowerShell Module is not available")
+)
+
+func init() {
+	var err error
+	powershell, err = exec.LookPath("powershell.exe")
+	if err != nil {
+		exit.WithError("%v", ErrPowerShellNotFound)
+	}
+}
+
+func powershellCmdOut(args ...string) (string, error) {
+	args = append([]string{"-NoProfile", "-NonInteractive"}, args...)
+	cmd := exec.Command(powershell, args...)
+	glog.Infof("[executing ==>] : %v %v", powershell, strings.Join(args, " "))
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	err := cmd.Run()
+	glog.Infof("[stdout =====>] : %s", stdout.String())
+	glog.Infof("[stderr =====>] : %s", stderr.String())
+	return stdout.String(), err
+}
+
+func powershellCmd(args ...string) error {
+	_, err := powershellCmdOut(args...)
+	return err
+}
+
+func parseLines(stdout string) []string {
```

Comment:
I see the usage of `ParseLines` function. This is not just trimming space but rather provides the entire output line by line in a way it can be iterated over. I would prefer keeping this function as it is. What do you think?